target_postgres/tests/test_standard_target.py

```diff
@@ -179,7 +179,7 @@ def test_port_config():
 
     engine: sqlalchemy.engine.Engine = connector._engine
     assert (
-        str(engine.url)
+        engine.url.render_as_string(hide_password=False)
```

Comment:
SQLA 2.0 changed the default behavior of str(engine.url) to obfuscate the password (probably smart). This alternative overrides that and shows the password so it can be matched to the expected connection string.